util/monoclock.c

```diff
@@ -19,7 +19,7 @@
 
 /**
  * monoclock_get(tv):
- * Store the current time in ${tv}.  If CLOCK_MONOTONIC is available, use
+ * Store in ${tv} the current time.  If CLOCK_MONOTONIC is available, use
```

Comment:
Nah, it just feels clumsy here.  It's a bit clumsy in the other function too, but there we need it for clarity.